### PR TITLE
Add a new api from_tflite to improve user experience.

### PR DESCRIPTION
**Pull request by fatcat-z**
1. Add a new api from_tflite() so users could call it to import a tflite model.
2. Add relative test code as well.

Fix https://github.com/onnx/tensorflow-onnx/issues/1938

Signed-off-by: Jay Zhang <jiz@microsoft.com>

### CODE REVIEW

**Review by hwangdeyu — COMMENTED** [review]
Add the description of new api `from_tflite` in https://github.com/onnx/tensorflow-onnx#python-api-reference?

File: tests/test_api.py [hunk 1]
```diff
@@ -232,5 +232,18 @@ def test_graphdef(self):
         self.assertAllClose([2.1193342], oy[0], rtol=0.1, atol=0.1)
 
 
+    def test_tflite(self):
```
Comment (hwangdeyu) [inline tests/test_api.py:235]: Should we also check the test `@check_tf_min_version("1.15")`, like the other API test?

**Comment by fatcat-z** [comment]
> Add the description of new api `from_tflite` in https://github.com/onnx/tensorflow-onnx#python-api-reference at the same time?

Good catch!

**Review by hwangdeyu — APPROVED** [review]
LGTM!

**Pull request merged**